src/common/uilocale.cpp

```diff
@@ -752,9 +752,14 @@ const wxLanguageInfo* wxUILocale::FindLanguageInfo(const wxString& locale)
     // to the entry description in the language database.
     // The locale string may have the form "language[_region][.codeset]".
     // We ignore the "codeset" part here.
+    wxString myLocale = locale;
```

Comment:
This is a pretty minor issue, but I'd rather rename `locale` function argument to something like `localeIn` and then call its local copy `locale` in order to avoid gratuitous diffs below that just rename the variable.

Reply:
Ok, I'll do that.